auxx/src/Command/Tx.hs

```diff
@@ -211,6 +217,18 @@ send sendActions idx outputs = do
             pure $ maybe (error "Unknown wallet address") (^. wusRootKey) (_userSecret ^. usWallet)
         | otherwise = (!! idx) <$> getSecretKeysPlain
 
+sendWithoutWitnesses
+    :: forall m. MonadAuxxMode m
+    => SendActions m
+    -> TxOut -> m ()
+sendWithoutWitnesses sendActions output = do
+  let fakeTxIn = TxInUtxo (unsafeHash ()) 1
+      fakeTx = UnsafeTx (fakeTxIn:|[]) (output:|[]) def
+      fakeTxAux = TxAux fakeTx V.empty
+
+  CmdCtx {ccPeers} <- getCmdCtx
+  void $ submitTxRaw (immediateConcurrentConversations sendActions ccPeers) fakeTxAux
```

Comment:
Hm, I think it makes more sense to make exactly 1 intentional error in 1 command.
`send-with-fake-input` sends a transaction with fake input, ok.
And then `send-without-witness` sends a transaction with fake input and without witnesses. We shouldn't rely on the fact that witnesses correctness is checked before input correctness (moreover, I guess it's not the case). And if input correctness is checked first, this command will fail the same way as `send-with-fake-input`.

P. S. Also it probably would be better to use invalid witness instead of using no witnesses at all.